app/core/src/main/java/com/fsck/k9/message/TextBodyBuilder.java

```diff
@@ -189,15 +187,15 @@ private String getSignature() {
 
     private String getSignatureHtml() {
         String signature = "";
-        if (!TextUtils.isEmpty(mSignature)) {
-            signature = textToHtmlFragment("\r\n" + mSignature);
+        if (!isEmpty(mSignature)) {
+            signature = "<div style='white-space: pre-wrap'>" + textToHtmlFragment("\r\n" + mSignature) + "</div>";
```

Comment:
Since we're wrapping the signature in a `<div>` tag don't need to add a line break in front of the signature.

```suggestion
            signature = "<div style='white-space: pre-wrap'>" + textToHtmlFragment(mSignature) + "</div>";
```

Reply:
Done